doc/source/configuration/wazuh.rst

```diff
@@ -0,0 +1,435 @@
++                 +-----------+----------------+------------------------------------------------+
+|                 | 514       | TCP (optional) | Wazuh Syslog collector (disabled by default)   |
++                 +-----------+----------------+------------------------------------------------+
+|                 | 55000     | TCP            | Wazuh server RESTful API                       |
++-----------------+-----------+----------------+------------------------------------------------+
+|                 | 9200      | TCP            | Wazuh indexer RESTful API                      |
++ Wazuh indexer   +-----------+----------------+------------------------------------------------+
+|                 | 9300-9400 | TCP            | Wazuh indexer cluster communication            |
++-----------------+-----------+----------------+------------------------------------------------+
+| Wazuh dashboard | 443       | TCP            | Wazuh web user interface                       |
++-----------------+-----------+----------------+------------------------------------------------+
+
+
+
+Make sure group mappings for wazuh-manager are added to the inventory ``etc/kayobe/inventory/groups``:
+
+.. code-block:: console
+
+  # Infra VM groups.
+
+  [wazuh:children]
+  wazuh-manager
+
+
+  [wazuh-manager]
+  # Empty group to provide declaration of wazuh-manager group.
+
+
+Add hosts group mappings to the inventory ``etc/kayobe/inventory/hosts``:
+
+.. code-block:: console
+
+  [wazuh-manager]
+    <wazuh.vm.name>
+
+
+Manually provisioned VM
+-----------------------
+
+In case where you can’t use infra-vms to deploy your wazuh-manager but you want to configure
+host using kayobe, there are some tips (note that depending on your setup this don’t have to always apply):
+
+* Depending on preferences host have to be part of some group in inventory. ``infra vm`` group still seems as best choice
+  You can use ``kayobe infra vm host configure`` to configure host in this case.
+  Bellow tips are based on assumption that infra-vm will be used.
+* user ``stack`` with password less sudo and accessible with ssh keys needs to be present on host.
+  It can be achieved in many different ways, depending on your setup.
+* lvm configuration should be placed in ``host_vars/<host_name>``
+* wazuh-manager host have to be part of ``infra-vms`` group (directly or as child)
+* network used on host needs to be defined in ``networks.yml`` and
+  if you have pre-alocated IP, it should be added to ``network-allocation.yml``. For example:
+
+
+``networks.yml``:
+
+.. code-block:: console
+
+    undercloud_admin_net_cidr: 10.10.224.0/24
+    undercloud_admin_net_allocation_pool_start: 10.10.224.3
+    undercloud_admin_net_allocation_pool_end: 10.10.224.200
+    undercloud_admin_net_gateway: 10.10.224.254
+
+
+``network-allocation.yml``:
+
+.. code-block:: console
+
+    undercloud_admin_net_ips:
+      nesmetprd01: 10.10.224.5
```

Comment:
provision_oc_net to be consistent with above examples?

Reply:
The point was to demonstrate that this network isn't one of the "standard" kayo networks. I will add clarification.